lib/ansible/cli/galaxy.py

```diff
@@ -1380,6 +1384,7 @@ def _execute_install_collection(
             allow_pre_release=allow_pre_release,
             artifacts_manager=artifacts_manager,
             disable_gpg_verify=disable_gpg_verify,
+            offline=context.CLIARGS.get('offline', False),
```

Comment:
Doesn't it always have a default per the arg declaration above? Is this fallback really needed, then?
```suggestion
            offline=context.CLIARGS['offline'],
```

Reply:
No, because of https://github.com/ansible/ansible/pull/78673#discussion_r960727518.